src/server/Document/component.jsx

```diff
@@ -84,25 +87,34 @@ const Document = ({
             {AMP_ANALYTICS_JS}
           </>
         )}
-      </head>
-      <body {...ampGeoPendingAttrs}>
-        <div id="root" dangerouslySetInnerHTML={{ __html: html }} />
         {scriptsAllowed && (
           <script
+            id="simorgh-envvars"
             dangerouslySetInnerHTML={{
-              __html: `window.SIMORGH_DATA=${serialisedData}`,
+              // Read env variables from the server and expose them to the client
+              __html: `window.SIMORGH_ENV_VARS=${appEnvVariables}`,
```

Comment:
Could follow that approach, although it doesn't seem to get rid of `dangerouslySetInnerHTML`? Looks like it just reads the text content rather than the `window`.